tests/test_alerter.py

```diff
@@ -416,12 +419,9 @@ class TestMessageBuilding(unittest.TestCase):
     def setUp(self):
         self.test_alerter = alerter.Alerter()
         self.freeze_time_value = "2020-03-10 09:00"
-        self.tz = os.environ.get("TZ", "local")
```

Comment:
I don't think the fallback ("local") was working as expected here and in other spots. I also don't think it's necessary if we force MST for these.